networking/openshift_network_security/logging-network-security.adoc

```diff
@@ -1,48 +1,32 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="logging-network-policy"]
-= Logging for egress firewall and network policy rules
+[id="logging-network-security"]
+= Audit logging for network security
 include::_attributes/common-attributes.adoc[]
-:context: logging-network-policy
+:context: logging-network-security
 
 toc::[]
 
-As a cluster administrator, you can configure audit logging for your cluster and enable logging for one or more namespaces. {product-title} produces audit logs for both egress firewalls and network policies.
+The OVN-Kubernetes network plugin uses Open Virtual Network (OVN) access control list (ACL) to manage `AdminNetworkPolicy`, `BaselineAdminNetworkPolicy`, `Networkpolicy`, and egress firewalls objects. Audit logging exposes `allow` and `deny` ACL events for `NetworkPolicy` and `BaselineAdminNetworkPolicy` custom resources (CR). Logging also exposes `allow`, `deny`, and `pass` ACL events for `AdminNetworkPolicy` (ANP) CR.
```

Comment:
```suggestion
The OVN-Kubernetes network plugin uses Open Virtual Network (OVN) access control list (ACL) to manage `AdminNetworkPolicy`, `BaselineAdminNetworkPolicy`, `Networkpolicy`, and `EgressFirewall` objects. Audit logging exposes `allow` and `deny` ACL events for `NetworkPolicy` and `BaselineAdminNetworkPolicy` custom resources (CR). Logging also exposes `allow`, `deny`, and `pass` ACL events for `AdminNetworkPolicy` (ANP) CR.
```
and what about for EFW? what kind of logging action is exposed?

Reply:
Looking at our docs I _think_ it is the same as Netpol `allow` and `deny`. I've updated accordingly. https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/logging-network-policy.html#nw-networkpolicy-audit-concept_logging-network-policy